django_mongodb/schema.py

```diff
@@ -0,0 +1,69 @@
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+
+
+class DatabaseSchemaEditor(BaseDatabaseSchemaEditor):
```

Comment:
Does not at the moment implement and schema migrations. We could add support for field renames and type conversions here, if we'd have use for that.